code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/local/LocalAuthConfigManager.java

```diff
@@ -0,0 +1,70 @@
+package io.cattle.platform.iaas.api.auth.integration.local;
+
+import io.cattle.platform.core.constants.AccountConstants;
+import io.cattle.platform.core.constants.ProjectConstants;
+import io.cattle.platform.core.model.Account;
+import io.cattle.platform.iaas.api.auth.SecurityConstants;
+import io.cattle.platform.iaas.api.auth.SettingsUtils;
+import io.cattle.platform.iaas.api.auth.dao.AuthDao;
+import io.cattle.platform.util.type.CollectionUtils;
+import io.github.ibuildthecloud.gdapi.factory.SchemaFactory;
+import io.github.ibuildthecloud.gdapi.model.ListOptions;
+import io.github.ibuildthecloud.gdapi.request.ApiRequest;
+import io.github.ibuildthecloud.gdapi.request.resource.impl.AbstractNoOpResourceManager;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class LocalAuthConfigManager extends AbstractNoOpResourceManager {
+
+    @Inject
+    private AuthDao authDao;
+
+    @Inject
+    SettingsUtils settingsUtils;
+
+    @Override
+    public Class<?>[] getTypeClasses() {
+        return new Class<?>[] {LocalAuthConfig.class};
+    }
+
+    @SuppressWarnings("unchecked")
+    @Override
+    protected Object createInternal(String type, ApiRequest request) {
+        if (!StringUtils.equalsIgnoreCase(LocalAuthConstants.CONFIG, request.getType())) {
+            return null;
+        }
+        Map<String, Object> config = CollectionUtils.toMap(request.getRequestObject());
+        String username = (String) config.get("username");
```

Comment:
FYI, if LocalAuthConfig was an interface you could just do `request.proxyRequestObject(LocalAuthConfig.class)` and then you can just do `goo.getName()` and not have to be casting all these strings and hardcoding fields names.